app/code/community/Ebizmarts/MailChimp/Model/Api/Subscribers.php

```diff
@@ -578,6 +428,288 @@ protected function addError($errorMessage)
         Mage::getSingleton('core/session')->addError($errorMessage);
     }
 
+    /**
+     * @param $customAtt
+     * @param $customer
+     * @param $mergeVars
+     * @param $key
+     * @param $helper
+     * @param $subscriberEmail
+     * @param $storeId
+     * @return array
+     */
+    protected function customizedAttributes($customAtt, $customer, $mergeVars, $key, $helper, $subscriberEmail, $storeId)
```

Comment:
https://travis-ci.org/mailchimp/mc-magento/jobs/545910306#L3236

441 | WARNING | [ ] Function's cyclomatic complexity (24) exceeds
     |         |     allowed maximum of 20
 441 | WARNING | [ ] The method parameter $helper is never used
 441 | WARNING | [ ] Line exceeds 120 characters; contains 121